lib/Graph/Graph.cpp

```diff
@@ -1838,6 +1838,22 @@ Function *Function::clone(llvm::StringRef newName,
   return newF;
 }
 
+/// Verify the input \p idx of a node \p N. Check that the node \p N is in the
+/// use-list of the corresponding input node.
+static void verifyNodeInput(const Node &N, size_t idx) {
+  auto input = N.getNthInput(idx);
+  auto refN = input.getNode();
```

Comment:
do we have any standard on a format of `auto` with the pointers? (would be good to stick to a common style).

Reply:
You mean we should use `auto *` here? Yeah. I wish the tooling could warn us in cases like `auto *` or `auto &` if we forget it.

Reply:
yeah, like auto * in this case.